OneCommunityGlobal/HighestGoodNetworkApp
# Jay - PERMISSION: Manage Admin Link Permission

# Description
This PR enables the user with the Edit Link Permission to be able to edit other users' links on their profile pages. 

## Related PRS (if any):
This frontend PR is related to the [#688](https://github.com/OneCommunityGlobal/HGNRest/pull/688)  backend PR.

…

## Main changes explained:
- Added conditional logic so that Edit button next to LINKS is visible to those with the permission.
- Granted permission to those with 'manageAdminLinks' permission.
…

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log as owner user
5. go to Other Links -> Permissions Management -> Manage User Permissions -> Choose a user without user management permissions such as volunteer or core team -> add Manage Admin Links in User Profile permission
6. verify that user is now able to see other users' links by clicking on Edit button next to LINKS (it's below the profile picture)

## Screenshots or videos of changes:


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/118868270/17846e35-966a-432d-a56d-36b2f7373d40


## Note:
**Unmute the video.**
Also, please test the personal links as well. They should also work.
